examples/medplum-chart-demo/src/components/Allergies.tsx

```diff
@@ -16,6 +16,7 @@ export function Allergies(props: AllergiesProps): JSX.Element {
   const { patient, encounter } = props;
   const [allergies, setAllergies] = useState<AllergyIntolerance[]>(props.allergies);
   const [opened, { open, close }] = useDisclosure(false);
+  const [codingValueInput, setCodingValueInput] = useState<Coding>();
```

Comment:
I'd call this `code` or `allergyCode`.

We try to avoid `value` as a suffix.

When using `Input` as a suffix, it typically means the variable is an actual "input" such as an `HtmlInputElement` or whatever.

All in all, this should probably be:

```ts
const [code, setCode] = useState<CodeableConcept>();
```